common/trinary/trit_tryte.c

```diff
@@ -18,6 +18,42 @@ static const trit_t TRYTE_TO_TRITS_MAPPINGS[TRYTE_SPACE]
                                            [NUMBER_OF_TRITS_IN_A_TRYTE] = {
                                                TRITS_TO_TRYTES_MAP};
 
+size_t num_trytes_for_trits(size_t num_trits) {
+  return (num_trits + NUMBER_OF_TRITS_IN_A_TRYTE - 1) /
+         NUMBER_OF_TRITS_IN_A_TRYTE;
+}
+
+trit_t get_trit_at(tryte_t *const trytes, size_t const length, size_t index) {
+  size_t tindex = index / 3U;
```

Comment:
move:
`+  size_t tindex = index / 3U;
+  if (tindex >= length) {
+    return 0;
+  }
+  tryte_t tryte = trytes[tindex];
+  size_t cindex = tryte == '9' ? 0 : tryte - '@';`
to a static function?